### PR TITLE
Remove uswds-gulp dependency and troubleshoot gulp

### PR DESCRIPTION
**Pull request by thisisdano**
This PR removes `uswds-gulp` as a dependency and includes everything that `uswds-gulp` provides locally:

- Adds a `.browserslistrc` file to include the autoprefixer browser support
- Hardcodes the path to the `uswds` package into the gulpfile

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Supported browsers
```
Comment (thisisdano) [inline .browserslistrc:1]: This is what autoprefixer uses to calculate its prefixes

File: gulpfile.js [hunk 2]
```diff
@@ -12,15 +12,14 @@ USWDS SASS GULPFILE
 */
 
 const autoprefixer = require("autoprefixer");
-const autoprefixerOptions = require("./node_modules/uswds-gulp/config/browsers");
```
Comment (thisisdano) [inline gulpfile.js:15]: Removed. Using `.browserslistrc` instead

File: gulpfile.js [hunk 3]
```diff
@@ -12,15 +12,14 @@ USWDS SASS GULPFILE
...
 const csso = require("postcss-csso");
 const gulp = require("gulp");
 const pkg = require("./node_modules/uswds/package.json");
 const postcss = require("gulp-postcss");
 const replace = require("gulp-replace");
 const sass = require("gulp-sass");
 const sourcemaps = require("gulp-sourcemaps");
-const uswds = require("./node_modules/uswds-gulp/config/uswds");
```
Comment (thisisdano) [inline gulpfile.js:23]: Hardcoded this path instead of relying on `uswds-gulp` for it

File: gulpfile.js [hunk 4]
```diff
@@ -81,28 +80,25 @@ gulp.task("copy-uswds-js", () => {
   return gulp.src(`${uswds}/js/**/**`).pipe(gulp.dest(`${JS_DEST}`));
 });
 
-gulp.task("build-sass", function(done) {
+gulp.task("build-sass", function (done) {
```
Comment (thisisdano) [inline gulpfile.js:83]: All these other changes are style changes enforced by running Prettier

File: gulpfile.js [hunk 5]
```diff
@@ -81,28 +80,25 @@ gulp.task("copy-uswds-js", () => {
...
   var plugins = [
     // Autoprefix
-    autoprefixer(autoprefixerOptions),
+    autoprefixer(),
```
Comment (thisisdano) [inline gulpfile.js:86]: Removed the options in favor of `.browserslistrc`

File: package.json [hunk 6]
```diff
@@ -29,7 +29,6 @@
     "gulp-sourcemaps": "^2.6.5",
     "postcss-csso": "^4.0.0",
     "sass": "^1.26.2",
-    "uswds": "^2.5.1",
-    "uswds-gulp": "github:uswds/uswds-gulp"
```
Comment (thisisdano) [inline package.json:33]: Removed dependency

**Review by dawnpm — APPROVED** [review]
thank you!

**Pull request merged**